README.md

```diff
@@ -66,7 +66,7 @@ Then check "Initialize this repository with a README". Under ".gitignore" select
 
 Now you have a Git repository on GitHub! Donc professionnel!
 
-We'll want to "clone" a copy of your repository to your local computer so you can use it. So click the "Clone or Download" button, then click the clipboard (icon) button to copy the URL to the clipboard.
+We'll want to "clone" a copy of your repository to your local computer so you can use it. So click the "Clone or Download" button, then click the clipboard (icon) button to copy the URL to the clipboard. It's easier to use clone with HTTPS so make sure this is selected. 
```

Comment:
Again, no. I am not telling the reader which way to do this. That's up to them. So, "You may find cloning via HTTPS easier" might be better. That said, I _always_ clone with SSL, if possible, and have never had a problem with it, so I'm not sure I agree with this. I'll adapt.